apiCalls.js

```diff
@@ -0,0 +1,37 @@
+const fetch = require("node-fetch");
+
+const movieFetch = async (table) => {
+  console.log(table)
```

Comment:
Make sure we get rid of this console log!

Reply:
😱